greenlight/lib/api/keys.py

```diff
@@ -14,6 +14,7 @@ def __init__(self, marionette_getter):
 
     @property
     def ACCEL(self):
-        if self.marionette_getter().session_capabilities['platformName'] == 'DARWIN':
+        if self.marionette_getter().session_capabilities['platformName'] == \
+                'DARWIN':
```

Comment:
Here and elsewhere please use implicit line continuations via parentheses for long expressions. Pep8 suggests as much, but I guess it isn't included in the linter.